draft-add-requirements.md

```diff
@@ -104,92 +108,137 @@ the IETF may publish, such as DNS-over-QUIC {{?I-D.ietf-dprive-dnsoquic}}.
 
 Do53: Unencrypted DNS over UDP port 53, or TCP port 53 {{?RFC1035}}.
 
-Equivalent: See {{equivalence}}.
+Designated: See {{designation}}.
+
+Designator: The network or resolver that issued the designation.
 
 
-# Use case description
+# Overall use case
 
 It is often the case that a client possesses no specific configuration for how to operate DNS, and at some point
-joins a network that it has no previous knowledge about. In such a case the usual existing behaviour is to
-dynamically discover the network's recommended Do53 resolver and use it. This long-standing practice works in
-nearly all networks, but presents a number of privacy and security risks that were the motivation for the development
-of encrypted DNS.
+joins a network that it cannot authenticate. It may have no prior knowledge of the network, or it may have connected
+previously to a network that looked the same. In either case the usual behaviour, because of lack of specific
+configuration, is to dynamically discover the network's recommended Do53 resolver and use it. This long-standing practice
```

Comment:
The aim here is to say that (a) we are concentrating on discovery in untrusted/unauthenticated networks, and (b) that sometimes this is the first connection to a new network, but (c) other times you might be connecting to a network that you've joined 500 times before. In the case (c) the network might look the same, but it might not be actually the same. We cannot be sure, so it's untrusted.

Tiru is working on a draft for trusted networks including IPSEC VPN.

Reply:
Sure. What I was trying to say is that the text - as I read it - seems to imply a relation between the networks we consider - untrusted network (a) (b) and (c) - and dynamic discovery. In fact ipsec vpn also have similar dynamic discovery.   I am fine if that is only me.

Reply:
Yes I can see that trusted networks might also use dynamic discovery. But since the scope of this document is untrusted networks, I think it's best to not say anything about what trusted networks might do. I'll leave it unchanged.